plexapi/base.py

```diff
@@ -393,20 +535,22 @@ def reload(self, key=None, **kwargs):
         """
         return self._reload(key=key, **kwargs)
 
-    def _reload(self, key=None, _overwriteNone=True, **kwargs):
+    def _reload(self, key: Optional[str] = None, _overwriteNone: bool = True, **kwargs: Any):
         """ Perform the actual reload. """
         details_key = self._buildDetailsKey(**kwargs) if kwargs else self._details_key
         key = key or details_key or self.key
         if not key:
             raise Unsupported('Cannot reload an object not built from a URL.')
         self._initpath = key
         data = self._server.query(key)
+        if data is None:
+            raise NotFound(f'Unable to find elem: {key=}')
```

Comment:
Is this necessary?

Reply:
as long as query could return `None`, handling of `None` should be done, it is done here as next step requires subscripting `data`

` self._loadData(data[0])`